core/src/main/scala/org/apache/pekko/persistence/postgres/db/ExtendedPostgresProfile.scala

```diff
@@ -13,7 +13,6 @@ trait ExtendedPostgresProfile
     with PgSearchSupport
     with PgNetSupport
     with PgLTreeSupport
-    with PgJsonSupport
```

Comment:
looks ok to me, the only place where we use Json column is for metadata, and we rely on `io.circe.Json`.